templates/medical/blood_donation.txt

```diff
@@ -0,0 +1,46 @@
+[center][b]Blood Donation Application Form[/b][/center]
+[hr]
+First time donor? [field] [small](yes/no)[/small][br]
+[hr]
+[u]For Repeat Donor[/u][br]
+Did you encounter any problems in your last donation? [field]
+[small](No problems / fainting / bruise / difficulties finding vein / other)[/small][br]
+[hr]
+Blood group: [field][br]
+Age: [field][i]year[/i] Sex: [field] Weight: [field][i]kg[/i][br]
+If more than 60-70 years old, Do you have medical certificate? [field] [small](yes/no)[/small][br]
+Occupation: [field][br]
+Name: [i]Mr. / Ms. / Mrs.[/i] [field] Signature: [field][br]
+[hr]
+[hr]
+[center][b]For your own safety and the safety of the patient who will receive your blood, please answer the following questions, if applicable, to the best of your knowledge:[/b][/center]
+[hr]
+[u]Category 1 [small](For women only)[/small][/u][br]
+1. Pregnant? [field][br]
+2. Do breast-feed? [field][br]
+3. Gave birth or miscarriage in the last 6 months? [field][br]
+[hr]
+[u]Category 2[/u]
+4. Had diarrhea in the last 7 days? [field][br]
+5. Had major surgery in the last 6 months or minor surgery in the last 7 days? [field][br]
+6. Do you drink alcohol or others? [field][br]
+7. Had a history of drug use or had you been improsoned in the last 3 years? [field][br]
+8. Had a blood transfusion in the past 1 year? [field][br]
+[u]Category 3[/u]
+9. Have menstruation? [small](to be answered by female only)[/small] [field][br]
+10. Do you feel fit enough and have enough rest last night? [field][br]
+11. Did you take any medicine / antibiotics? [field][br]
+12. Have you or any of your family member ever had hepatitis? [field][br]
+13. Have asthma, epilepsy, chronic skin disease, chronic cough, tuberculosis, allergies, high blood pressure, heart/kidney/thyroid disease, cancer, bleeding disorder, etc? [field][br]
+14. Have ear/body piercings, tattoos made or removed or acupuncture? [field][br]
+15. Did you get any vaccinations in the last 2 months? [field][br]
+26. Others: [field][br]
+[hr]
+[small]I hereby certify that I have answered the following questions truthfully and that, to the best of my knowledge, my blood is safe for donation.
+I have been informed that my blood will be tested for syphilis, hepatitis B and C, as well as HIV/AIDS.
+I hereby voluntarily donate blood without expecting any type of remuneration.
+The blood may be given to any patient or for research purpose as deemed suitable by Torch medical personnel.
+I certify that the Torch medical staff are not responsible for any untoward effects that may occour after this blood donation.
+I shall be pleased to donate blood again.[/small]
+Donor signature: [field][br]
+Doctor/Staff signature: [field][br]
```

Comment:
Not certain that the nurse involved needs to sign, this form may only need the donor's informed consent